src/packages/end-to-end/README.md

```diff
@@ -0,0 +1,20 @@
+# Graphweaver End to End Test Suite
+
+This package tests the following:
+
+- Using the CLI to create a new GraphWeaver app
+- Imports a data source to be used to query
+- Makes requests to the API and tests the results
+
+## Database
+
+The database used to run these tests can be found here:
+
+https://github.com/lerocha/chinook-database
+
+The SQLite file has been copied from the above directory and stored in `database.sqlite`.
```

Comment:
Is it icky to store a db as a file in a project? I've never encountered keeping the db in the project before but it could be fine b/c its just an example.

Reply:
This is ok for this SQLite databases are files 👍